hooks/boot/01-helpers.rb

```diff
@@ -10,7 +10,7 @@ def module_enabled?(kafo, name)
       mod.enabled?
     end
 
-    def log_and_say(level, message)
+    def log_and_say(level, message, log_only = false)
```

Comment:
Would it be better to have a ```log(level, message)``` and ```say(level, message)``` where ```log_and_say(level, message)``` composes them? They don't appear to share variables.